bodhi/tests/client/test___init__.py

```diff
@@ -392,6 +392,6 @@ def test_url_flag(self, send_request, query):
                 'close_bugs': True, 'stable_karma': None, 'csrf_token': 'a_csrf_token',
                 'staging': False, 'builds': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'autokarma': True,
                 'edited': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'suggest': None,
-                'notes': u'this is an edited note', 'request': None, 'bugs': u'',
-                'unstable_karma': None, 'type': 'bugfix'})
+                'notes': u'this is an edited note', 'notes_file': None,
+                'request': None, 'bugs': u'', 'unstable_karma': None, 'type': 'bugfix'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
```

Comment:
Hi, @bowlofeggs , I do need help getting started. Actually, I think I need plenty of help. I seem to have been running in circles and thus made 0 headway since your initial comments a couple of days ago. First, would I also place these unit tests in /bodhi/tests/client/test__init__.py in the class TestEdit? 

I think my biggest problem is that I'm confused as to how to mock a file, or whether I even need to be mocking a file. I wrote [a simple unit test](https://gist.github.com/crungehottman/e6cd109242502e7a02c5a91aa289cf03) that entailed the three tests you mentioned above, but I don't know how to incorporate this tests into the bodhi project codebase. I'm not even sure if this is how I should be testing --notes-file... I'd prefer to be testing it the way @cverna implemented TestEdit -- BUT -- I made a copy of cverna's test, replacing --notes with --notes-file, but for some reason way beyond me, that causes cverna's test to fail (ha! how on earth did I manage that?) at assert_called_with. I'm sure it has something to do with my not knowing how to test this using a file. Should I be testing this with essentially the same test cvernas created (TestEdit.test_url_flag), except replacing --notes with --notes-file, and somehow creating a --notes-file to test? Also, I don't understand why everyone in the class is naming their tests test_url_flag and testing 

> self.assertEqual(bindings_client.base_url, 'http://localhost:6543')

in addition to testing assert_called_with.

Is [this](http://click.pocoo.org/5/testing/#file-system-isolation) something I should be looking at re: testing a --notes-file, or am I way off?